### DISCUSSION TITLE
How to get inventory information for more than 50skus?

### DISCUSSION THREAD

**Discussion by qhangfan** [saleweaver/python-amazon-sp-api#637] [Q&A]
![image](https://user-images.githubusercontent.com/109368774/179274254-abad8ede-cd7d-4a4a-b3ac-036a2500f243.png)
Thank you for developing such a useful SDK.
When I request it with the Inventories().get_inventory_summary_marketplace method, the returned nextToken value is null. How can I get nextToken?
Also I'm using Windows10 + python3.9 + pycharm, I can't find the file credentials.yml, In which directory should I create a new one? could you help me? thank you very much!

**Comment by zyzjian** [comment]
report--report type:GET_FBA_MYI_UNSUPPRESSED_INVENTORY_DATA
